### ISSUE TITLE
Command Hyper does not work

### ISSUE THREAD

**Issue by Frying7415** [wet-sandwich/hyper.nvim#33]
Script which loads Hyper expects non null opts, so it'd be nice to include `opts = {}` in the installation example like this:
```
{
  'wet-sandwich/hyper.nvim',
  dependencies = { 'nvim-lua/plenary.nvim' },
  opts = {},
}
```
If opts are not passed lua raises an exception about missing opts when you try to run `:Hyper`.

**Comment by wet-sandwich** [comment]
Good catch, I thought I had that covered. I assume you're running the latest version from main and not a tagged release? I have some stuff in the works in main that hasn't been released yet and that is causing this issue. You might want to switch to a tagged version.

**Comment by Frying7415** [comment]
@wet-sandwich I was following the Installation guide in readme, so I suppose Lazy pulled the latest main indeed. Will try switching to latest tagged and check the issue there, thanks!
UPD: Not reproducible after switching to latest release. Would be nice to add `version = "*"` to the Installation section in readme as well :)
